special-interest-groups/sig-usability/README.md

```diff
@@ -0,0 +1,21 @@
+# Usability Special Interest Group(Usability SIG)
+
+Usability SIG covers the usability issues about all of the TiDB products. Our work includes
```

Comment:
This sounds like a cross-sig thing. And it doesn't own any SIG, right? Then I think this should be a Working Group, instead of a SIG.

Reply:
Agree, I also think working group is more suitable.

Reply:
we will have own code ASAP, this is a real interest group dedicated to discussing anything related to tidb usability improvements. If a specific improvement is identified, then a working group is set up to complete it

Reply:
same with https://github.com/kubernetes/community/tree/master/sig-usability